Amplify/Categories/Storage/StorageCategory+ClientBehavior.swift

```diff
@@ -10,46 +10,60 @@ import Foundation
 extension StorageCategory: StorageCategoryBehavior {
 
     @discardableResult
-    public func getURL(key: String,
-                       options: StorageGetURLOperation.Request.Options? = nil) async throws -> URL {
+    public func getURL(
+        key: String,
+        options: StorageGetURLOperation.Request.Options? = nil
+    ) async throws -> URL {
         try await plugin.getURL(key: key, options: options)
     }
 
     @discardableResult
-    public func downloadData(key: String,
-                             options: StorageDownloadDataOperation.Request.Options? = nil) async throws -> StorageDownloadDataTask {
-        try await plugin.downloadData(key: key, options: options)
+    public func downloadData(
+        key: String,
+        options: StorageDownloadDataOperation.Request.Options? = nil
+    ) -> StorageDownloadDataTask {
+        plugin.downloadData(key: key, options: options)
     }
 
     @discardableResult
-    public func downloadFile(key: String,
-                             local: URL,
-                             options: StorageDownloadFileOperation.Request.Options?) async throws -> StorageDownloadFileTask {
-        try await plugin.downloadFile(key: key, local: local, options: options)
+    public func downloadFile(
+        key: String,
+        local: URL,
+        options: StorageDownloadFileOperation.Request.Options?
```

Comment:
Would this be a good time to add a default `= nil` to `options` as well?

Same for the `AWSS3StoragePlugin+AsyncClientBehavior.swift` file

Reply:
Changed